### DISCUSSION TITLE
organizzazione fisica - 9

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#88] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/b7b66ba1-c00f-4f71-8f5b-71314c55ce2d)

**Comment by 7Andrei** [comment]
![OrganizzazioneFisica9](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97444723/c7576387-f10c-4e69-b7d3-26ba522ef1ec)
